### PR TITLE
Update terms of use

### PR DESCRIPTION
**Pull request by talithachin**
updated terms of use to latest, version based on 15 sep 2021

### CODE REVIEW

File: faq.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
       rel="stylesheet"
     />
     <link rel="stylesheet" type="text/css" href="style.css" />
-    <title>RedeemSG - Track Vouchers Easily</title>
+    <title>RedeemSG - Voucher System for the Singapore Government</title>
```
Comment (charmzzyyy) [inline faq.html:20]: Should we change the meta description too?

File: terms-of-use.html [hunk 2]
```diff
@@ -1,695 +1,696 @@
...
-        <div class="row">
-          <div class="col-12">
-            <div class="masthead-content">
-              <img src="img/sg-crest.svg" class="sg-crest" />
-              <p>A Singapore Government Website</p>
-            </div>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta name="description" content="RedeemSG - Track Redemptions Easily" />
```
Comment (charmzzyyy) [inline terms-of-use.html:7]: Similar comment to update meta description

File: terms-of-use.html [hunk 3]
```diff
@@ -1,695 +1,696 @@
...
-                    impliedly require these Terms of Use to be altered such that
-                    the Third Party Terms are complied with, the parties herein
-                    agree that the Terms of Use shall be deemed to be so altered
-                    but only to the extent necessary for compliance.
-                  </div>
+              </div>
+              <div class="clause-set">
+                <div class="clause-numbering">4.7</div>
+                <div class="clause-text">
+                  4.7. Without prejudice and in addition to the foregoing, GovTech shall not be responsible for your use
```
Comment (charmzzyyy) [inline terms-of-use.html:423]: Remove "4.7" in body copy as there's already clause-numbering

File: faq.html (hunk 1 above)
Comment (talithachin) [reply faq.html:20]: ok, yes good to change the meta description

**Comment by sheikhshack** [comment]
There is still usages of `Track vouchers easily` on some parts of the page. Cool if I changed them all to `Voucher System for the Singapore Government`?

File: terms-of-use.html [hunk 4]
```diff
@@ -1,695 +1,696 @@
...
-                    Party, such direct rights of enforcement against you.
-                  </div>
+              <div class="spacing-sm"></div>
+              <div class="clause-set">
+                <div class="clause-numbering"><strong>5</strong></div>
+                <div class="clause-text">
+                  <strong>
+                    Your Consent to Access Functions of Your Device
+                  </strong>
                 </div>
```
[the remark this answers is not in the document]
Comment (sheikhshack) [reply terms-of-use.html:440]: Might need to check the deployed version again to confirm if this is addressed. I removed the `<strong>` tags and replaced content with this. Let me know if this is what you have in mind

image

**Pull request merged**